Tokamak Native/NSAppDelegate.swift

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019-2020 Tokamak contributors
```

Comment:
How's about renaming the directory to `TokamakDemo Native` as it only compiles the demo code?